Assets/Mirror/Runtime/NetworkManager.cs

```diff
@@ -391,6 +410,9 @@ public bool StartServer()
         {
             InitializeSingleton();
 
+            authenticator.OnStartServer();
```

Comment:
we should do this after runInBackground. runInBackground seems to be more important here.